public/components/datasources/components/manage/data_connection.tsx

```diff
@@ -89,6 +96,10 @@ export const DataConnection = (props: { dataSource: string }) => {
   const [refreshIntegrationsFlag, setRefreshIntegrationsFlag] = useState(false);
   const refreshInstances = () => setRefreshIntegrationsFlag((prev) => !prev);
 
+  useEffect(() => {
+    setIsS3ConnectionWithLakeFormation(checkIsConnectionWithLakeFormation(datasourceDetails));
+  }, [datasourceDetails]);
+
   useEffect(() => {
     const searchDataSourcePattern = new RegExp(
       `flint_${escapeRegExp(datasourceDetails.name)}_default_.*`
```

Comment:
Since the PR moves away from always using `default` as the database, this search pattern should be widened to `flint_${escapeRegExp(datasourceDetails.name)}_.*`

Reply:
The indexpattern in the creation request as well: ``indexPattern: `flint_${config.connectionDataSource}_default_${config.connectionTableName}__*`,`` in `public/components/integrations/components/setup_integration.tsx`

Reply:
Fixed these